workflow-examples/src/test/java/com/redhat/parodos/examples/integration/SimpleWorkFlow.java

```diff
@@ -0,0 +1,175 @@
+package com.redhat.parodos.examples.integration;
+
+import com.redhat.parodos.examples.integration.utils.ExamplesUtils;
+import com.redhat.parodos.sdk.api.ApiClient;
+import com.redhat.parodos.sdk.api.ApiException;
+import com.redhat.parodos.sdk.api.Configuration;
+import com.redhat.parodos.sdk.api.ProjectApi;
+import com.redhat.parodos.sdk.api.WorkflowApi;
+import com.redhat.parodos.sdk.api.WorkflowDefinitionApi;
+import com.redhat.parodos.sdk.model.ArgumentRequestDTO;
+import com.redhat.parodos.sdk.model.ProjectRequestDTO;
+import com.redhat.parodos.sdk.model.ProjectResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowDefinitionResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowRequestDTO;
+import com.redhat.parodos.sdk.model.WorkFlowResponseDTO;
+import com.redhat.parodos.sdk.model.WorkFlowResponseDTO.WorkStatusEnum;
+import com.redhat.parodos.sdk.model.WorkRequestDTO;
+import com.redhat.parodos.workflow.consts.WorkFlowConstants;
+import com.redhat.parodos.workflow.enums.WorkFlowProcessingType;
+import com.redhat.parodos.workflow.enums.WorkFlowType;
+import com.redhat.parodos.workflow.enums.WorkType;
+import com.redhat.parodos.workflow.utils.CredUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.util.Strings;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * @author Gloria Ciavarrini (Github: gciavarrini)
+ */
+@Slf4j
+public class SimpleWorkFlow {
+
+	private static final String projectName = "project-1";
+
+	private static final String projectDescription = "an example project";
+
+	@Before
+	public void setUp() throws IOException {
+
+	}
+
+	@Test
+	public void runSimpleWorkFlow() {
+		log.info("Running simple flow");
+		ApiClient apiClient = Configuration.getDefaultApiClient();
+		apiClient.addDefaultHeader(HttpHeaders.AUTHORIZATION, "Basic " + CredUtils.getBase64Creds("test", "test"));
+
+		ProjectApi projectApi = new ProjectApi(apiClient);
+		try {
+			ExamplesUtils.waitProjectStart(projectApi);
+		}
+		catch (ApiException e) {
+			fail(e.getMessage());
+		}
+		log.info("Project is ✔️ on {}", apiClient.getBasePath());
+
+		ProjectResponseDTO testProject;
+
+		try {
+			// RETRIEVE ALL PROJECTS AVAILABLE
+			log.info("Get all available projects");
+			List<ProjectResponseDTO> projects = projectApi.getProjects();
+
+			// CHECK IF testProject ALREADY EXISTS
+			testProject = projects.stream()
```

Comment:
What do you think about extracting this to a method? It is used here and in line 103.